Net/src/HTTPServerResponseImpl.cpp

```diff
@@ -126,9 +126,22 @@ void HTTPServerResponseImpl::sendFile(const std::string& path, const std::string
 	{
 		_pStream = new HTTPHeaderOutputStream(_session);
 		write(*_pStream);
+#ifdef POCO_USE_SENDFILE_FOR_HTTPSERVER
 		if (_pRequest && _pRequest->getMethod() != HTTPRequest::HTTP_HEAD)
 		{
+			delete _pStream; // delete the stream to flush the HTTP headers to the socket, required by HTTP 1.0 and above
```

Comment:
Nicer would be first just call flush(). Not very good habit to flush with destructor.

Reply:
Thank you. I replace destructor with ```flush()```